functions/src/offers/index.ts

```diff
@@ -6,9 +6,10 @@ import { Change, EventContext } from 'firebase-functions/lib/cloud-functions';
 import { IOffer, Offer, OfferStatus } from '../models/offers';
 import DocumentSnapshot = firestore.DocumentSnapshot;
 
-const queueStatusUpdateTriggers = (
-  change: Change<DocumentSnapshot>,
-): Promise<void[]> => {
+// Init the admin db
+const db = firestore();
```

Comment:
This initialization is not what I was talking about
You need to add `admin.initializeApp();` before you initialize `firestore`

Reply:
oh ok makes sense.